storage/src/test/java/tech/pegasys/artemis/storage/ChainStorageClientTest.java

```diff
@@ -46,17 +46,18 @@
 class ChainStorageClientTest {
 
   private static final BeaconState INITIAL_STATE =
-      DataStructureUtil.randomBeaconState(UnsignedLong.ZERO, 1);
+      new DataStructureUtil(3).randomBeaconState(UnsignedLong.ZERO);
```

Comment:
why do we need a different DataStructureUtil here?

Reply:
Because it's being used in a static context.  I'm trying to avoid having static `DataStructureUtil` instances as then the order of test execution will vary what the seed is and give different results.